views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -134,8 +133,26 @@ define([
          */
         function getInputLevel() {
             var sum;
+            var frequencyLength;
+            var getRndInteger;
 
             analyser.getByteFrequencyData(frequencyArray);
+            frequencyLength = frequencyArray.length;
+
+            if (frequencyArray.length && /(iPhone|iPad)/i.test(navigator.userAgent)) {
```

Comment:
It would be better to put the check in a proper helper, to ease the maintenance.
Moreover, global variables should be namespaced using `window.`
```javascript
function isMobileDevice() {
    return  /(iPhone|iPad)/i.test(window.navigator.userAgent);
}